pkg/arvo/app/dojo.hoon

```diff
@@ -532,24 +532,24 @@
       ?:  ?=([%show %3] -.mad)
         (dy-rash %tan (dy-show-source q.mad) ~)
       ?:  ?=(%brev -.mad)
+        ?:  ?=($?(%eny %now %our) p.mad)
+          (dy-rash %tan [(crip "{(scow %tas p.mad)} is immutable")] ~)
```

Comment:
fyi, it's perfectly acceptable (and in fact, more common) to use `$?`'s shorthand, `?(%etc)`. But `$?` is also fine!

The `[]` here (and below) are superfluous though.

Lastly, both the `+crip` and `+scow` calls here are a bit off. `(scow %tas %something)` is equivalent to `(trip %something)`, and the latter is usually preferred. And if you're going to produce a `@t` tank instead of a `[%leaf tape]` one, you probably want to directly concatenate the two: `(cat 3 p.mad ' is immutable')`.  
(For more complex interpolations, of course, `leaf+"some {(com plex)} string"` is still more ergonomic and preferred.)

(To be clear, this is not a big deal, I'm just nit-picking style here and trying to convey nuances.)

```suggestion
        ?:  ?=(?(%eny %now %our) p.mad)
          (dy-rash %tan (cat 3 p.mad ' is immutable') ~)
```